Build error for v-analyzer in "V Apps and Modules" CI workflow

In "V Apps and Modules" CI workflow (defined in `.github/workflows/v_apps_and_modules_compile_ci.yml`), **builf of `v-analyzer` fails** (step "Build vlang/v-analyzer"):
```sh
[ERROR] Failed to build v-analyzer
✓ Prepared output directory `./bin` .
> Failed build_thirdparty_obj_file cmd
>           mod: builtin
>          path: /home/runner/work/v/v/thirdparty/libbacktrace/backtrace.o
>   source_file: /home/runner/work/v/v/thirdparty/libbacktrace/backtrace.c
> wd before cmd: /tmp/v-analyzer
> getwd for cmd: /home/runner/work/v/v
>           cmd: '/home/runner/work/v/v/thirdparty/tcc/tcc.exe' -std=c99 -D_DEFAULT_SOURCE   -Wno-misleading-indentation -Wno-jump-misses-init -Wno-error=jump-misses-init -Wno-typedef-redefinition -fwrapv -g  -Wno-misleading-indentation -Wno-jump-misses-init -Wno-error=jump-misses-init -Wno-typedef-redefinition -fPIC -D GC_THREADS=1 -D GC_BUILTIN_ATOMIC=1 -I "/home/runner/work/v/v/thirdparty/libgc/include" -I "/home/runner/work/v/v/thirdparty/libbacktrace" -o "/home/runner/.vmodules/.cache/1a/1aec18c6c0e875fe4ff6e28b2428059b.module.builtin.o" -c "/home/runner/work/v/v/thirdparty/libbacktrace/backtrace.c"
builder error: In file included from /home/runner/work/v/v/thirdparty/libbacktrace/backtrace.c:142:
/home/runner/work/v/v/thirdparty/libbacktrace/base.c:5880: error: include file 'unwind.h' not found
```

See for example this run with this error => https://github.com/vlang/v/actions/runs/23897538962/job/69685793179

The include file `unwind.h` is not present and needed by `libbacktrace` (updated recently in commit e78481994a892734d00a24a16951a2df75f76631).

The `unwind.h` include file is present in Ubuntu package `libunwind-18-dev` => proposal to add it in build depends for this workflow.
> [!NOTE]
> You can use the 👍 reaction to increase the issue's priority for developers.
>
> Please note that only the 👍 reaction to the issue itself counts as a vote.
> Other reactions and those to comments will not be taken into account.

--- a/.github/workflows/v_apps_and_modules_compile_ci.yml
+++ b/.github/workflows/v_apps_and_modules_compile_ci.yml
@@ -43,6 +43,7 @@ jobs:
             v retry -- sudo apt -qq install libgc-dev libsodium-dev libssl-dev sqlite3 libsqlite3-dev
             v retry -- sudo apt -qq install libfreetype6-dev libxi-dev libxcursor-dev libgl-dev libxrandr-dev libasound2-dev xfonts-75dpi xfonts-base libegl-dev
             v retry -- sudo apt -qq install sassc libgit2-dev # needed by gitly
+            v retry -- sudo apt -qq install libunwind-18-dev # needed by v-analyzer for libbacktrace
           else
             v retry brew install sassc libgit2
           fi

